modules/compute/src/main/java/org/apache/ignite/internal/compute/IgniteComputeImpl.java

```diff
@@ -204,6 +207,7 @@ public <T, R> CompletableFuture<BroadcastExecution<R>> submitAsync(
                                             entry.getValue(),
                                             entry.getKey(),
                                             table.tableId(),
+                                            table.zoneId(),
```

Comment:
In order tobe consistent, I'd rather add zoneId param prior to tableId.

Reply:
:ok_hand: